nvaccess/nvda
# Add pronunciation of Mac Option key symbol "⌥"

### Link to issue number:
n/a (minor/trivial change)

### Summary of the issue:
NVDA doesn't announce the symbol that macs use to represent the mac "Option" key ("⌥") (see [Mac keyboard shortcuts documentation](https://support.apple.com/en-us/HT201236)).

### Description of user facing changes
Adds a symbol description for this symbol ("mac Option key" announced at "none" level and up, similar to the existing entry for "mac Command key" at the "none" level for symbol "⌘").

### Description of development approach
Followed the ["Defining Symbol Information" developer documentation](https://www.nvaccess.org/files/nvda/documentation/developerGuide.html#toc10), using the existing "mac Command key" entry and similar recent PR #14548 as examples

### Testing strategy:

Didn't attempt to adjust any automated tests (assumed they were already present generally for symbol pronunciation). Manually verified reading by:

1. `runnvda.bat` from this branch
1. Browse to [Mac keyboard shortcuts docs](https://support.apple.com/en-us/HT201236)
1. Read through the content under the "Mac keyboard shortcuts" H1, particularly the list item with content "Option (or Alt) ⌥"
1. Verify that the symbol is read out as "mac Option key" as expected
1. Change NVDA's "Settings > Speech > Punctuation/symbol level" to "none" and repeat above steps

### Known issues with pull request:
Mac commonly uses a few other symbols to represent keyboard modifiers (eg, when displaying keyboard shortcut hints), per the [Mac keyboard shortcuts documentation](https://support.apple.com/en-us/HT201236). I've left the other cases alone because they either:

* Already have a comparable entry ("Command ⌘")
* Are usually represented as an image, not a character symbol ("Fn <img src="https://support.apple.com/library/content/dam/edam/applecare/images/en_US/mac/mac-fn-key-globe-icon.png" width="16" height="16" alt="globe icon">")
* Already have more general symbol names that would be applicable in more contexts ("Shift ⇧", "Caps Lock ⇪", "Control ^")

### Change log entries:
```
New features
- Added pronunciation of Mac Option key symbol "⌥". (#14682)
```

### Code Review Checklist:

  - description is up to date
  - change log entries
  - Unit tests
  - System (end to end) tests
  - Manual testing
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
